updater/cmd/updater/main.go

```diff
@@ -60,11 +74,35 @@ func main() {
 
 	if err := installer.Install(rb); err != nil {
 		log.Default().Printf("Failed to install: %s", err)
-		// TODO: Change packagestatuses to have error message
+
+		// Set the state to failed before rollback so collector knows it failed
+		if setErr := monitor.SetState(packagestate.DefaultFileName, protobufs.PackageStatus_InstallFailed, err); setErr != nil {
```

Comment:
@cpheps it would be great if you could fill out the error message too so the collector has that to send to the Server as well.

Reply:
This does fill out the error message because the error passed in is not nil. Unless I'm missing what you're saying.

Reply:
nope, i'm just dumb and totally missed that